src/python/pants/backend/python/framework/django/dependency_inference_test.py

```diff
@@ -42,101 +37,77 @@ def rule_runner() -> RuleRunner:
             *pex.rules(),
             *dependency_inference.rules(),
             *detect_apps.rules(),
-            QueryRule(ParsedPythonDependencies, [ParsePythonDependenciesRequest]),
+            *core_rules(),
+            QueryRule(InferredDependencies, [dependency_inference.InferDjangoDependencies]),
         ],
         target_types=[PythonSourceTarget],
     )
 
 
-def assert_deps_parsed(
-    rule_runner: RuleRunner,
-    content: str,
-    *,
-    constraints: str,
-    expected_imports: dict[str, ImpInfo] | None = None,
-    expected_assets: list[str] | None = None,
-    filename: str = "path/to/app0/migrations/0001_initial.py",
-) -> None:
-    expected_imports = expected_imports or {}
-    expected_assets = expected_assets or []
-    rule_runner.set_options(
-        [
-            "--no-python-infer-string-imports",
-            "--no-python-infer-assets",
-        ],
-        env_inherit={"PATH", "PYENV_ROOT", "HOME"},
-    )
+def do_test_migration_dependencies(rule_runner: RuleRunner, constraints: str) -> None:
```

Comment:
For future reference, I would have preferred as few changes to the test as possible, as it is the insurance policy that the underlying code change is good, and so keeping it stable is strong reassurance that that test hasn't been cooked to match, so to speak.

Reply:
+1. I did look at that and tried to see what the test changes where coming from.. but gave up ;)

Reply:
Sorry, this round-tripped through some code that assumed that we'd do Py3 only